### PR TITLE
fix debian help

### PR DESCRIPTION
**Pull request by ZhiShengYuan**

### CODE REVIEW

**Review by ZenithalHourlyRate — CHANGES_REQUESTED** [review]
```diff
-has_backports: '' ,has_nfw: 'non-free-firmware' 
+has_backports: '' , has_nfw: ' non-free-firmware' 
```
Code style

```diff
-non-free {{has_nfw}}
+non-free{{has_nfw}}
```

The extra whitespace should be added in `has_nfw`

**Comment by ZenithalHourlyRate** [comment]
Also please make these changes to https://github.com/mirrorz-org/mirrorz-help/blob/master/contents/debian-security.mdx

The style of the commit message should be consistent with other commit messages.

**Comment by ZhiShengYuan** [comment]
> Also please make these changes to https://github.com/mirrorz-org/mirrorz-help/blob/master/contents/debian-security.mdx
> 
> The style of the commit message should be consistent with other commit messages.

I noticed that there were no spaces before the `,` , so I reformatted the code according to the original format

File: contents/debian.mdx [hunk 1]
```diff
@@ -20,13 +20,13 @@ Debian Buster 以上版本默认支持 HTTPS 源。如果遇到无法拉取 HTTP
...
       title: 'Debian 版本',
       items: [
-        ['Debian 11 (bullseye)', { release_name: 'bullseye', security: '-security', is_sid: '', has_backports: '' }],
-        ['Debian 12 (bookworm)', { release_name: 'bookworm', security: '-security', is_sid: '', has_backports: '' }],
-        ['sid', { release_name: 'sid', security: '-security', is_sid: '# ', has_backports: '# ' }],
-        ['testing', { release_name: 'testing', security: '-security', is_sid: '', has_backports: '' }],
-        ['Debian 10 (buster)', { release_name: 'buster', security: '/updates', is_sid: '', has_backports: '' }],
-        ['Debian 9 (stretch)', { release_name: 'stretch', security: '/updates', is_sid: '', has_backports: '' }],
-        ['Debian 8 (jessie)', { release_name: 'jessie', security: '/updates', is_sid: '', has_backports: '# ' }]
+        ['Debian 11 (bullseye)', { release_name: 'bullseye', security: '-security', is_sid: '' , has_backports: '', has_nfw: '' }],
```
Comment (ZenithalHourlyRate) [inline contents/debian.mdx:23]:
```diff
-is_sid: '' , has_backports: '', has_nfw: ''
+is_sid: '', has_backports: '', has_nfw: ''
```

**Review by ZenithalHourlyRate — APPROVED** [review]

**Pull request merged**